src/global-firestore-data/overridden-fulfillment-choices.ts

```diff
@@ -134,3 +124,39 @@ export const addArbitraryOptIn = (
       },
     })
   );
+
+/**
+ * Precondition: the requirement id lists are disjoint
+ */
+export const resolveConflicts = (
+  courseUniqueID: string | number,
+  naturallyFulfilledRequirementIds: string[],
+  arbitraryOptInRequirementIds: string[],
+  optOutRequirementIds: string[]
```

Comment:
I think it's better to pass these in as an options object

Reply:
Can you be more specific @zachary0kent?